src/_guides/libraries/c-interop.md

```diff
@@ -7,7 +7,8 @@ sqlite: "https://github.com/dart-lang/sdk/tree/master/samples/ffi/sqlite"
 ---
 
 Dart mobile, command-line, and server apps running on the [Dart Native
-platform](/overview#platform/) can use the dart:ffi library to call native C APIs.
+platform](/overview#platform/) can use the dart:ffi library to call native C APIs,
+and to access and manipulate native memory.
```

Comment:
_access_ is a bit unclear (https://developers.google.com/style/word-list#letter-a), and I'm not sure what you mean by _manipulate_. This is publishable, but I'd prefer to be more specific. Would this be accurate:

```suggestion
and to read, write, allocate, and deallocate native memory.
```